torch/_functorch/_aot_autograd/autograd_cache.py

```diff
@@ -388,11 +393,56 @@ def post_compile(self, result: TOut, fx_config: _CompileFxKwargs) -> TOut:
         ...
 
 
+@dataclass
+class CompiledFxGraphLoadable(InductorOutput[CompiledFxGraph]):
+    """
+    A full compiled fx graph that doesn't need to lookup the FxGraphCache
+    to run
+    """
+
+    result: CompiledFxGraph
```

Comment:
As discussed offline, I think right now we can start from serializing the inductor artifacts with pickle directly. Later we need a way to map inductor files directly onto disk.